### PR TITLE
combine integrity with authenticity and remove integrity as a separat…

### LINKED ISSUE #94: The use of 'authenticity' and 'integrity'
(1) The use of 'authenticity' and 'integrity'

There was an old debate around which variant of definitions to use. The TSP spec eventually decided to use 'authenticity' to mean BOTH authentic identifier and integrity of the message. This definition IS used in the tech arch as well in the "Design Goals" section. That's good. BUT, later in the "Messages" section, it listed "Integrity" separately from "authenticity". I believe we should stick to the expansive definition of 'Authenticity' throughout, and merge 'Integrity' to 'Authenticity' and remove 'Integrity' in the later list. This modification will make this spec consistent with itself and with the TSP spec.

If such discrepancy exists in the terms wiki, we should make some changes there too. (Of course we could note there that we have chosen this way while others may use them differently so the readers are aware.)

### PR DESCRIPTION
**Pull request by wenjing**
Resolves #94

### CODE REVIEW

**Comment by wenjing** [comment]
Also resolves #95 #96 #97

File: spec/spec.md [hunk 1]
```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
 
 VIDs can be divided into two subclasses as shown in figure 9:
```
Comment (andorsk) [inline spec/spec.md:310]: Suggestion: Define "external system"

File: spec/spec.md [hunk 2]
```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
...
+Currently, the best-known form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] TODO-REFERENCE. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. As of mid-2024, there are almost 200 DID methods registered in the W3C DID Spec Registry, many, if not all, of which can be classified as either a SCID or an XVID.
```
Comment (andorsk) [inline spec/spec.md:314]:
```suggestion
Currently, the best-known form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] `TODO-REFERENCE`. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. As of mid-2024, there are almost 200 DID methods registered in the W3C DID Spec Registry, many, if not all, of which can be classified as either a SCID or an XVID.
```
highlighting to make it easier to spot for later.

**Review by andorsk — APPROVED** [review]
a few nits, but approved.

File: spec/spec.md (hunk 2 above)
Comment (andorsk) [inline spec/spec.md:314]:
```suggestion
Currently, the most common form of VID is a [[xref: toip, decentralized identifier]] (DID) as defined by the [W3C Decentralized Identifiers (DID) V1.0 Specification] TODO-REFERENCE. In general, DIDs fulfill the requirement of Design Principle #4 ([Decentralization by Design and Default](https://trustoverip.org/permalink/Design-Principles-for-the-ToIP-Stack-V1.0-2022-11-17.pdf)). The W3C DID specification defines a generic syntax for DIDs and a standard data model for a [[xref: toip, DID document]]—the artifact obtained through [[xref: toip, DID resolution]] that contains the [[xref: toip, cryptographic keys]] and [[xref: toip, service endpoints]] bound to the DID. The syntax for a specific type of DID and the process for creating, reading, updating, or deactivating the associated DID document is defined by a [[xref: toip, DID method]]. 
```
Sentence isn't required.

File: spec/spec.md [hunk 3]
```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
 
 VIDs can be divided into two subclasses as shown in figure 9:
 
-  1. **[[xref: toip, Self-certifying identifiers]] (SCIDs)** are cryptographically bound to the original key pair used to generate the VID. This means the binding between the VID and the key pair can be verified purely using cryptography—without reference to any external system. SCIDs have the advantage of being highly secure (as strong as the cryptographic algorithm used), maximally decentralized (because no external system is required), and fully portable (any external system can be used as a [[xref: toip, witness]], and the [[xref: toip, VID controller]] can change witnesses at will).
+  1. **[[xref: toip, Self-certifying identifiers]] (SCIDs)** are cryptographically bound to the original key pair used to generate the VID and subsequent key material where key rotation is supported. This means the binding between the VID and the key material can be verified purely using cryptography—without reference to any external system. SCIDs have the advantage of being highly secure (as strong as the cryptographic algorithm used), decentralized (because no external system is required), and portable.
```
Comment (andorsk) [inline spec/spec.md:311]: maybe not for now, but if you're going to put the advantages, why not the disadvantages?

File: spec/spec.md (hunk 1 above)
Comment (wenjing) [reply spec/spec.md:310]: I'm deferring all VID subclass related issues to a later PR as needed.

File: spec/spec.md (hunk 2 above)
Comment (wenjing) [reply spec/spec.md]: There are a lot of these

File: spec/spec.md (hunk 3 above)
Comment (wenjing) [reply spec/spec.md:311]: Again deferring

**Pull request merged**